lib/cher/error.go

```diff
@@ -177,14 +177,18 @@ func WrapIfNotCher(err error, msg string) error {
 
 // WrapIfNotCherCodes will wrap an error unless it is a cher with specific codes.
 func WrapIfNotCherCodes(err error, msg string, codes []string) error {
-	if err == nil {
-		return nil
-	}
+	return WrapIfNotCherCode(err, msg, codes...)
+}
 
+func WrapIfNotCherCode(err error, msg string, codes ...string) error {
 	var cErr E
 	if errors.As(err, &cErr) && slicecontains.String(codes, cErr.Code) {
```

Comment:
`slicecontains` is probably from before this was available in the standard library, but now that it is could we use `slices.Contains` instead?

```suggestion
	if errors.As(err, &cErr) && slices.Contains(codes, cErr.Code) {
```